packages/jest-matchers/package.json

```diff
@@ -12,5 +12,10 @@
     "jest-matcher-utils": "^19.0.0",
     "jest-message-util": "^19.0.0",
     "jest-regex-util": "^19.0.0"
+  },
+  "devDependencies": {
+    "ansi-regex": "^2.1.1",
```

Comment:
Because I use them in here: https://github.com/facebook/jest/pull/3119/files#diff-5d7e608b44e8080e7491366117a2025fR20
They're also already available through `chalk`